theme.js

```diff
@@ -2647,15 +2647,8 @@ export default (variables = defaultThemeVariables) => ({
     '.full-screen': {
       title: {
         container: {
-          // We want the title background gradient to be
-          // visible underneath the navigation bar, but the
-          // title text should be rendered below the
-          // navigation bar.
```

Comment:
Did you verify this background gradient behavior after your changes or just the `ImageGalleryOverlay`?

Reply:
Honestly, I've seen no background gradient, ever. This seems like an old comment, while we had a different design.
I'll check it again to verify, I'm curious now

Reply:
I've reverted deleting this comment.
I see some styles I can't figure out, including gradient.
I see we're setting some background gradient for title and description, but it's not visible in my app.
I'll leave it as is, just in case.